src/edit_distance.jl

```diff
@@ -78,17 +81,83 @@ function edit_distance(G₁::AbstractGraph, G₂::AbstractGraph;
             vs = setdiff(1:nv(G₂), [v for (u, v) in λ])
 
             if k < nv(G₁) # there are still vertices to process in G₁?
+                # u be the next node to be transformed
+                u = k+1
+                # transformation operations on neighbor of u 
+                u_edited_neighbors = Vector{Tuple{Int,Int}}()
+                u_neighbors = all_neighbors(G₁, u)
```

Comment:
So this variable is not used anywhere? Can you also double check `v_neighbors`? Please reserve some time to carefully review your code before asking for new review. :+1:

Reply:
Okay, thanks for the review. I will carefully plan and review my code.